test/common/pdbs_exist.go

```diff
@@ -0,0 +1,49 @@
+package common
+
+import (
+	"testing"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	namespaces = []Namespace{
+		{
+			Name: "redhat-rhmi-rhsso",
+			PodDisruptionBudgetNames: []string{
+				"keycloak",
+			},
+		},
+		{
+			Name: "redhat-rhmi-user-sso",
+			PodDisruptionBudgetNames: []string{
+				"keycloak",
+			},
+		},
+		// {
```

Comment:
Adding these in but commented out for now until the new version of 3scale is included at the beginning of April